common/wuanalysis/anacommon.cpp

```diff
@@ -34,7 +34,13 @@ int PerformanceIssue::compareCost(const PerformanceIssue & other) const
 
 void PerformanceIssue::print() const
 {
-    printf("[%" I64F "dms] %s: %s\n", statUnits2msecs(cost), scope.str(), comment.str());
+    stat_type costsecs = statUnits2seconds(cost);
```

Comment:
Would the formatStatistic() function be useful here?  E.g. formatStatistic(out, cost, SMeasureTimeNs);